webrender/src/renderer.rs

```diff
@@ -4013,6 +4020,47 @@ impl Renderer {
         }
         self.device.end_frame();
     }
+
+    // Sets the blend mode. Blend is unconditionally set if the "show overdraw" debugging mode is
```

Comment:
shouldn't it only affect the final (screen) pass?

Reply:
Could you elaborate?

Reply:
We render a frame in multiple passes. All but the last one go into off-screen targets. Since we are only displaying the screen overdraw, we don't want the blending overrides to affect rendering in any other passes than the last one.

Reply:
OK, changed it.